index.html

```diff
@@ -1,96 +1,64 @@
-        			<p class="pt0 f6 fw1 white">Series editors: Rebecca Lave and Sara Wylie</p>
-        			<p class="pt0 f6 fw1 white">Series designers: James Beach, Kyala Shea, Lourdes Vera</p>
-                    <p class="pt0 f6 fw1 white">Website: Shaquilla Singh</p>
-                </div>
-    		</div>
-
-		<!-- INTRO BLURB -->
-		<div id="ch1" class="vh-50 w-50-l w-100-m w-100-ns dt tc tile pv4-ns">
-            <div class="tile-content">
-                <a href="epa-under-siege.html">
-                    <h3 class="f6 fw2 ttu">Part 1</h3>
-                    <h1 class="f1 fw2 ttu" style="color:#4FBBA9;">The EPA Under Siege</h1>
-                    <h2 class="f6 f5-l">Trump’s Assault in History and Testimony</h2>
-                    <h3 class="f4 fw2">An in-depth analysis of the Environmental Protection Agency (EPA) under Trump and Administrator Scott Pruitt.</h3>
-		    <div class="button">Read Part 1</div>
-                </a>
-
-            </div>
-		</div>
-        <div id="comp" class="vh-25 w-50-l w-100-m w-100-ns dt tc tile pv4-ns">
-            <div class="tile-content">
-                <a href="epa-under-siege.html#h.bgps8xwogn0q">
-                    <h3 class="f5 fw2 ttu white">Part 1</h3>
-                    <h1 class="f2 fw2 ttu white">Interview Compendium</h1>
-                    <h3 class="f4 fw2 white">Quotes from over fifty EDGI interviews with long-term federal agency employees</h3>
-                </a>
-            </div>
-        </div>
-        <div id="soon" class="vh-25 w-50-l w-100-m w-100-ns dt tc tile">
-            <div class="tile-content">
-                <h3 class="f6 fw2 ttu white">Further Parts</h3>
-                <h1 class="f3 fw2 ttu white">Check back soon for more</h1>
-            </div>
-		</div>
-
-
-
-        <!-- FOOTER -->
-        <footer class="dib w-100 mid-gray tc pt4 pb1" style="background-color:#f4f4f4">
-            <small class="f6 mt4">
-                <a rel="license" href="https://creativecommons.org/licenses/by-nc-sa/4.0/"><img class="pa2" alt="Creative Commons Licence" style="border-width:0" src="https://licensebuttons.net/l/by-nc-sa/4.0/80x15.png" /></a><br />This work is licensed under a <a rel="license" href="https://creativecommons.org/licenses/by-nc-sa/4.0/">Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License</a>.
-                <br>
-                <b><a href="https://envirodatagov.org"><p class="pt1">The Environmental Data &amp; Governance Initiative, 2017</p></a></b>
-            </small>
-        </footer>
-
+<body>
```

Comment:
@shaqsingh -- also wanted to chat with you about this but forgot (aii!) would it be a major issue to move the id/style stuff out of the body tags? I think <body> </body> should be moved to the layouts.